web_services/foxx/brightid/initdb.js

```diff
@@ -50,7 +51,8 @@ function createIndexes() {
   console.log("creating indexes ...");
   for (let index of indexes) {
     const coll = arango._collection(index.collection);
-    coll.ensureIndex({type: 'persistent', fields: index.fields})
+    delete index.collection;
```

Comment:
you refer to this deleted field in the console log two lines below

Reply:
It's fixed
https://github.com/BrightID/BrightID-Node/commit/b1c38e9343091e4d500a5a8796e31009fac2323a